src/content/changelog/4-74-0-2020-09-08.md

```diff
@@ -0,0 +1,27 @@
+---
+title: '4.72.0: 2020-09-08'
+date: 2020-09-08T04:00:00.000Z
+version: 4.74.0
+changelog:
+  - API
+---
+
+### Added
+
+* Added the *Reset Linode Root Password* ([POST /linode/instances/{linodeId}/password](/api/v4/linode-instances-linode-id-password/#post)) endpoint to the *Linode Instances* collection. Use this endpoint to reset the root password for a Linode.
```

Comment:
This is specifically for Bare Metal instances. I would check with @Dorthu to see if this should be announced.

Reply:
This endpoint is not specific to bare metal instances

Reply:
Not the endpoint, the password reset comment itself, which I believe was coming from this line in the changelog: https://bits.linode.com/LinodeAPI/apinext/pull/2910/files#diff-4ac32a78649ca5bdd8e0ba38b7006a1eR14